### PR TITLE
Check item_drop amount clientside

### PR DESCRIPTION
**Pull request by rubenwardy**

### CODE REVIEW

**Comment by rubenwardy** [comment]
> <@rubenwardy> only new clients connecting to a new server will be able to drop just one item
> <@rubenwardy> so old clients can only drop the old stack

File: src/game.cpp [hunk 1]
```diff
@@ -2655,10 +2655,12 @@ void Game::processItemSelection(u16 *new_playeritem)
 }
 
 
-void Game::dropSelectedItem()
+void Game::dropSelectedItem(bool single_item)
 {
 	IDropAction *a = new IDropAction();
 	a->count = 0;
```
Comment (nerzhul) [inline src/game.cpp]: use ternary operator ?

**Comment by nerzhul** [comment]
sounds okay, i prefer to have the sneak check client side, server side can be inaccurate

**Comment by rubenwardy** [comment]
@nerzhul done

**Pull request merged**